src/components/about/About.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import "./about.scss"
+
+const About = props => {
+  return (
+    <div className="outer-container">
+      <div className="about-container">
+        <h1>
+          <span id="about-us-header">ABOUT US</span>
+        </h1>
+        <p>
+          For entry level developers, people interested in starting a new career
+          in tech, and anyone wanting to encourage and help newer developers.
+          Join us to hear relevant speakers, have a slice of pizza, and make new
+          connections! We look forward to meeting you. If you’d like to be a
+          part of our slack channel, you can be a part of the conversation here:
```

Comment:
Let's capitalize "Slack". You might have seen Slack used in lowercase - especially in their logo - but looking at their website, it looks like they prefer it capitalized when used as an adjective in the middle of regular text like we're doing here.

![image](https://user-images.githubusercontent.com/11567955/79538736-c6802700-8039-11ea-9792-ef0b85a4e1a2.png)

Reply:
Capitalized "Slack"